sdk/compiler/damlc/lib/DA/Cli/Damlc.hs

```diff
@@ -824,6 +827,12 @@ execIde telemetry (Debug debug) enableScenarioService autorunAllScenarios option
               -- individual options. As a stopgap we ignore the argument to
               -- --jobs.
               , optThreads = 0
+              , optMbPackageName = pName <$> mPkgConfig
+              , optMbPackageVersion = mPkgConfig >>= pVersion
+              , optMbPackageConfigPath = Just pkgPath
+              -- Since the main package changes, we use GHC's default of `main` everywhere, since plumbing the actual unit id
```

Comment:
I think I understand this comment after a bit of though - we used to *not* set optMbPackageName/optMb... fields when running the IDE, so optUnitId would always return Nothing and all the IDE logic would essentially pretend packages don't exist for diagnostics (?) - now that we set those fields we need to tell the IDE to ignore them via the optHideUnitId field?

If that's the right interpretation, can we include a description of that in a bit more depth either here or on the field? The current comment here and on the field definition didn't help me understand very much.

Reply:
You've got it correct yes, I'll improve the comment